src/libraries/Common/tests/TestUtilities/TestUtilities.csproj

```diff
@@ -8,6 +8,7 @@
       and instead use runtime checks.
     -->
     <TargetFrameworks>$(NetCoreAppCurrent);net461</TargetFrameworks>
+    <DefineConstants Condition="'$(EnableAggressiveTrimming)' == 'true'">$(DefineConstants);BUILT_WITH_EAT</DefineConstants>
```

Comment:
Let's spell this out. Not everyone understands what `EAT` means.

Reply:
Changed how this is being done now, see [`6d44922` (#51697)](https://github.com/dotnet/runtime/pull/51697/commits/6d44922b92f9df6db107a9b2276f0474df1448fc) and [`d3e215a` (#51697)](https://github.com/dotnet/runtime/pull/51697/commits/d3e215a773af27b628b8213ae1b17228eab90d49)